root/frontend/src/routes/App.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+
+import Home from './Home';
+import Workouts from './Workouts';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
```

Comment:
Any particular reason to use `element={...}` here?

Reply:
This was the template given on the doc for react router v6.